core/src/main/java/org/hibernate/ogm/datastore/impl/AvailableDatastoreProvider.java

```diff
@@ -19,6 +19,7 @@
 	EHCACHE( "org.hibernate.ogm.datastore.ehcache.impl.EhcacheDatastoreProvider" ),
 	MONGODB( "org.hibernate.ogm.datastore.mongodb.impl.MongoDBDatastoreProvider" ),
 	FONGO( "org.hibernate.ogm.datastore.mongodb.impl.FongoDBDatastoreProvider" ),
+	NEO4J( "org.hibernate.ogm.datastore.neo4j.remote.impl.RemoteNeo4jDatastoreProvider" ),
```

Comment:
At the moment, I'm using NEO4J for the remote and NEO4J_EMBEDDED for the embedded with the assumption that neo4j remote is the most common use case. NEO4J is much nicer to use.